src/oslo_aero_coag.F90

```diff
@@ -639,13 +638,14 @@ subroutine clcoag(q, pmid, pdel, temperature, cldnum, cldfrc, delt_inverse, ncol
 
                    !process modes don't change mode except so4 condensate which becomes coagulate instead
                    !assumed to have same sink as MODE_IDX_OMBC_INTMIX_AIT
-                   if( .NOT. is_process_mode(l_index_donor,.true.)   &
-                        .OR. ( (l_index_donor.eq.chemistryIndex(l_so4_a1))  .AND. modeIndexCoagulator .eq. MODE_IDX_OMBC_INTMIX_COAT_AIT) ) then
+                   if( .NOT. is_process_mode(l_index_donor,.true.) .or.  &
+                        ( (l_index_donor == chemistryIndex(l_so4_a1)) .and. &
+                           modeIndexCoagulator == MODE_IDX_OMBC_INTMIX_COAT_AIT) ) then
```

Comment:
There are parentheses around the first equality check but not the second one. Something to fix someday?